bigquery/google/cloud/bigquery/client.py

```diff
@@ -670,7 +680,7 @@ def load_table_from_file(self, file_obj, destination,
                  be determined, or if the ``file_obj`` can be detected to be
                  a file opened in text mode.
         """
-        job_id = _make_job_id(job_id)
+        job_id = _make_job_id(job_id, prefix=job_id_prefix)
```

Comment:
(nit) I don't think you need to call this with keyword-arg syntax.

Reply:
Fixed with c0a7120